src/components/track/desktop/stats/Stats.tsx

```diff
@@ -80,9 +80,9 @@ const Stats = memo(
                 contentTitle={contentTitle}
                 // Map out all of the users so that the selector is cheaper inside the
                 // rendered component
-                userId1={slice[0] ? slice[0].user_id : undefined}
-                userId2={slice[1] ? slice[1].user_id : undefined}
-                userId3={slice[3] ? slice[2].user_id : undefined}
```

Comment:
hmm i know this is removed code but this line is weird `userId3={slice[3] ? slice[2].user_id : undefined}`. we're checking slice[3] and grabbing slice[2]'s user id. calling out to make sure that you're not missing anything in your refactor.

Reply:
Good callout, was wondering about this

Reply:
CC: @raymondjacobson was this intentional?